modules/default/updatenotification/node_helper.js

```diff
@@ -93,13 +93,17 @@ module.exports = NodeHelper.create({
 			delay = 60 * 1000;
 		}
 
-		var self = this;
+		let self = this;
```

Comment:
Can we get rid of this self?

Reply:
I'm not sure, as a Java developer I always hated that `this` is mutable in JavaScript which makes me do this overly defensive :)

Could change it if you think it's safe.

Reply:
One has to check always if it doesnt have side effects. But you can leave it for now and I might check it out later in another PR